src/node_modules/_file-explorer.js

```diff
@@ -0,0 +1,292 @@
+var setval = require('setval')
```

Comment:
So i copy & paste the code from setval to the project?

I still need the logic of turning an array of pathnames into a nested object and `setval` does only that. I can put it into `src/app/setval.js` maybe? Would that be good?